src/version.rs

```diff
@@ -0,0 +1,23 @@
+use std::env;
+
+// include version into the binary at compile time
+pub fn get_sdk_version() -> &'static str {
```

Comment:
I find it weird that this returns a string, in a public module.

Suggestions:
1) make the module pub(crate), to make this an internal concern
or 
2) make it return just the version number, keep it public, and prepend the unleash-client-rust bit within the relevant code.